pkg/exchange/binance/exchange.go

```diff
@@ -105,15 +93,10 @@ func New(key, secret string) *Exchange {
 	futuresClient.HTTPClient = binanceapi.DefaultHttpClient
 	futuresClient.Debug = viper.GetBool("debug-binance-futures-client")
 
-	if isBinanceUs() {
+	if util.IsBinanceUs() {
 		client.BaseURL = BinanceUSBaseURL
 	}
 
-	if paperTrade() {
-		client.BaseURL = BinanceTestBaseURL
-		futuresClient.BaseURL = FutureTestBaseURL
-	}
```

Comment:
this is still required

Reply:
no need

my new code is added before create new binance client as followings
```
        if util.IsPaperTrade() {
		binance.UseTestnet = true
	}
	var client = binance.NewClient(key, secret)
```
it'll help to set BaseURL of binance client to its test base url. I did run it and it works.